core/lib/types/src/storage/witness_block_state.rs

```diff
@@ -5,8 +5,43 @@ use serde::{Deserialize, Serialize};
 use crate::{StorageKey, StorageValue};
 
 /// Storage data used during Witness Generation.
-#[derive(Debug, Default, Serialize, Deserialize)]
+#[derive(Debug, Default, Clone)]
 pub struct WitnessBlockState {
```

Comment:
Let's avoid word "block" here as it can misinterpreted as "l2 block". I would name it "WitnessStorageState"